docs/videos/healthcare-and-life-sciences/Factor-Xa-example/submit.sh

```diff
@@ -7,9 +7,10 @@
 # Creative Commons Attribution 4.0 International (CC BY 4.0) License.
 # See <https://creativecommons.org/licenses/by/4.0/>.
 
-#SBATCH -N 2
-#SBATCH --ntasks-per-node 30
-#SBATCH --partition compute
+#SBATCH -N 1
+#SBATCH --ntasks-per-node 1
```

Comment:
so it doesn't really use the host CPUs?

Reply:
I will pass to @danielahlin to answer.

Reply:
As much as is currently possible is moved to the GPUs but the host CPUs are still used.